Documentation/README_ManipulationHandler.md

```diff
@@ -6,9 +6,14 @@ The *ManipulationHandler* script allows for an object to be made movable, scalab
 
 ## How to use the manipulation handler
 
-Add the [`ManipulationHandler.cs`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ManipulationHandler.cs) component to a GameObject.
+Add the `ManipulationHandler` script component to a GameObject:
 
-Make sure to also add a collider to the object, matching its grabbable bounds. To make the object respond to near articulated hand input, add the [`NearInteractionGrabbable.cs`](https://github.com/Microsoft/MixedRealityToolkit-Unity/blob/mrtk_release/Assets/MixedRealityToolkit.Services/InputSystem/NearInteractionGrabbable.cs) script as well.
+- Selecting the GameObject in the Hierarchy
+- In the Inspector, click the Add Component button
```

Comment:
same thing here - do we really need to explain how to add a component?

Reply:
removed